File: mvn-advent-of-code/src/test/java/advent/twenty_seventeen/Day25TheHaltingProblemTest.java
package advent.twenty_seventeen;

import advent.common.DailyProblem;
import org.junit.Test;

import static org.junit.Assert.*;

public class Day25TheHaltingProblemTest {
    @Test
    public void checkReal() {
        DailyProblem<Integer, Integer> dp = new Day25TheHaltingProblem("/twenty_seventeen/Day25-TheHaltingProblem-input.txt");

        assertEquals(2474, dp.getPart1Answer().intValue());
    }

    @Test
    public void checkExample1() {
        DailyProblem<Integer, Integer> dp = new Day25TheHaltingProblem("/twenty_seventeen/Day25-TheHaltingProblem-example#1.txt");

        assertEquals(3, dp.getPart1Answer().intValue());
    }
}

File: scala-advent-of-code/src/test/scala/advent/twenty_seventeen/Day25TheHaltingProblemTest.scala
package advent.twenty_seventeen

import org.scalatest.{FlatSpec, Matchers}

class Day25TheHaltingProblemTest  extends FlatSpec with Matchers {
  "2017 Day 25 - Input" should "calculate answers" in {
    val dp = new Day25TheHaltingProblem("/twenty_seventeen/Day25-TheHaltingProblem-input.txt")
    dp.part1Answer should be(0)
    dp.part2Answer should be(0)
  }

  "2015 Day 9 - Example #1" should "calculate answers" in {
    val dp = new Day25TheHaltingProblem("/twenty_seventeen/Day25-TheHaltingProblem-example#1.txt")
    dp.part1Answer should be(0)
    dp.part2Answer should be(0)
  }
}
